examples/ustream/ustream_server.c

```diff
@@ -131,7 +132,7 @@ int main(int argc, FAR char *argv[])
     }
 #endif
 
-  acceptsd = accept(listensd, (struct sockaddr*)&myaddr, &addrlen);
+  acceptsd = accept(listensd, (struct sockaddr *)&myaddr, &addrlen);
```

Comment:
```suggestion
  acceptsd = accept(listensd, (FAR struct sockaddr *)&myaddr, &addrlen);
```

Reply:
Since the whole file doesn't add FAR, I prefer to not add FAR. If you like, please add all FAR with new patch.